accounting/settings.py

```diff
@@ -47,27 +47,32 @@
 
 LOCAL_APPS = (
     'accounting.account',
+    'accounting.address',
     'accounting.authentication',
     'accounting.bank',
+    'accounting.common',
     'accounting.invoice',
     'accounting.registration',
+    'accounting.third_party_api',
 )
 
 INSTALLED_APPS = (
     'django.contrib.admin',
     'django.contrib.auth',
     'django.contrib.contenttypes',
     'django.contrib.sessions',
+    'django.contrib.sites',
     'django.contrib.staticfiles',
 
     'django_countries',
     'django_extensions',
     'djmoney',
     'huey.contrib.djhuey',
-    'localflavor',
     'rest_framework',
     'rest_framework.authtoken',
     'simple_email_confirmation',
+    'simple_history',
```

Comment:
This is nice - we only use it for changes made through the Django admin currently, right?

Reply:
Yes, but we have to actually [add the integration](https://django-simple-history.readthedocs.io/en/latest/usage.html#integration-with-django-admin), which I did not get around to doing just yet. I did add this right now though so that at least things are getting tracked in the meanwhile.

If the use case for this becomes important for admins, we can add an API endpoint for history by [querying history](https://django-simple-history.readthedocs.io/en/latest/usage.html#querying-history).

Reply:
You mean that the changes are being recorded in the DB when made via the admin, but that the history isn't actually visible from the admin until you add that integration? If so that's fine yep, this is more important to have for debugging than as a feature for the admin yep.

Reply:
Yep, that's the situation: it's being used in the Invoice model only right now, just not shown properly through the admin.